### PR TITLE
Improve docs for input data and solver installation

### PR DESCRIPTION
**Pull request by darrylmelander**

### CODE REVIEW

File: doc/OnlineDocs/source/tasks/install_solvers.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The choice of solver often comes down to cost. CBC is free, but is typically
+slower than commercial alternatives. Commercial solvers are faster, but
+require a license and may require additional configuration. Some solvers
+offer free licenses for academic or research use. Check with the appropriate
+vendor for details.
+
+CBC
+---
+CBC is a free, open-source linear solver. On Linux and Mac platforms, CBC
```
Comment (bknueven) [inline doc/OnlineDocs/source/tasks/install_solvers.rst:20]:
```suggestion
CBC is a free, open-source MILP solver. On Linux and Mac platforms, CBC
```

File: doc/OnlineDocs/source/tasks/install_solvers.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	conda install -c gurobi gurobi
+
+Gurobi requires a valid license. See the documentation on the 
+`Gurobi <https://gurobi.com>`_ website for instructions on how to
+acquire and install a license.
+
+Depending on your license, you may not be able to use the latest version
+of the solver. A specific release can be installed as follows::
+
+	conda install -c gurobi gurobi=8
```
Comment (bknueven) [inline doc/OnlineDocs/source/tasks/install_solvers.rst:46]: We should probably direct users to PyPI like Xpress below: https://pypi.org/project/gurobipy/

**Review by bknueven — COMMENTED** [review]
Looks pretty good. Just some minor comments.

Do you know anything about using HiGHS with Pyomo? That could be another open-source MILP solver we refer users to now..

**Comment by darrylmelander** [comment]
> Do you know anything about using HiGHS with Pyomo?

I know you can use it with the appsi solver interfaces, but I don't think it's available as a "normal" solver. And with appsi there are extra configuration steps that I'm not confident I'll get right. Maybe we should ask Michael B to document it a little more explicitly.

File: doc/OnlineDocs/source/tasks/install_solvers.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+CPLEX
+-----
+Instructions for installing Python bindings for CPLEX can be found
+`here <https://www.ibm.com/support/knowledgecenter/en/SSSA5P_12.8.0/ilog.odms.cplex.help/CPLEX/GettingStarted/topics/set_up/Python_setup.html>`_.
```
[the remark this answers is not in the document]
Comment (darrylmelander) [reply doc/OnlineDocs/source/tasks/install_solvers.rst:51]: That only gives you the trial edition of cplex. I updated the documentation though.

File: doc/OnlineDocs/source/tasks/install_solvers.rst (hunk 2 above)
Comment (darrylmelander) [reply doc/OnlineDocs/source/tasks/install_solvers.rst:46]: I added conda and pip (PyPI) instructions for solvers available from both sources.

**Pull request merged**